### PR TITLE
Additional configuration to the Azure SQL server deployment

### PR DESCRIPTION
**Pull request by qamago**

### CODE REVIEW

File: templates/sql-server.json [hunk 1]
```diff
@@ -2,36 +2,112 @@
     "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "sqlServerName": {
+        "environmentName": {
```
Comment (rivoric) [inline templates/sql-server.json]: Environment name is never used

File: templates/sql-server.json [hunk 2]
```diff
@@ -2,36 +2,112 @@
...
+                "administratorLogin": "[concat('d', uniqueString(resourceGroup().id, guid(parameters('sqlServerName'))))]",
+                "administratorLoginPassword": "[parameters('sqlServerAdminPassword')]"
+            },
+            "resources": [
+                {
+                    "apiVersion": "2015-05-01-preview",
+                    "type": "auditingSettings",
+                    "name": "[variables('AuditPolicyName')]",
+                    "dependsOn": [
+                        "[concat('Microsoft.Sql/servers/', parameters('sqlServerName'))]"
```
Comment (rivoric) [inline templates/sql-server.json]: Not needed as this appears within the SQL server resource block - dependancy is implicit

File: templates/sql-server.json [hunk 3]
```diff
@@ -2,36 +2,112 @@
...
+                        "storageAccountAccessKey": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('sharedARMStorageName')), providers('Microsoft.Storage', 'storageAccounts').apiVersions[0]).keys[0].value]",
+                        "retentionDays": "90"
+                    }
+                },
+                {
+                    "apiVersion": "2015-05-01-preview",
+                    "type": "securityAlertPolicies",
+                    "name": "[variables('SecurityAlertPolicyName')]",
+                    "dependsOn": [
+                        "[concat('Microsoft.Sql/servers/', parameters('sqlServerName'))]",
```
Comment (rivoric) [inline templates/sql-server.json]: First dependsOn not needed

File: templates/sql-server.json [hunk 4]
```diff
@@ -2,36 +2,112 @@
...
+                    "apiVersion": "2014-04-01-preview",
+                    "location": "[resourceGroup().location]",
+                    "properties": {
+                        "administratorType": "ActiveDirectory",
+                        "login": "[parameters('sqlServerActiveDirectoryAdminLogin')]",
+                        "sid": "[parameters('sqlServerActiveDirectoryAdminObjectId')]",
+                        "tenantId": "[subscription().tenantId]"
+                    },
+                    "dependsOn": [
+                        "[concat('Microsoft.Sql/servers/', parameters('sqlServerName'))]"
```
Comment (rivoric) [inline templates/sql-server.json]: Again not needed, see second comment

File: templates/sql-server.json (hunk 1 above)
Comment (qamago) [reply templates/sql-server.json]: Removed

File: templates/sql-server.json (hunk 2 above)
Comment (qamago) [reply templates/sql-server.json]: "[concat('Microsoft.Sql/servers/', parameters('sqlServerName'))]"
removed

File: templates/sql-server.json (hunk 3 above)
Comment (qamago) [reply templates/sql-server.json]: "[concat('Microsoft.Sql/servers/', parameters('sqlServerName'))]", removed

File: templates/sql-server.json (hunk 4 above)
Comment (qamago) [reply templates/sql-server.json]: "[concat('Microsoft.Sql/servers/', parameters('sqlServerName'))]", removed

File: templates/sql-server.json (hunk 3 above)
Comment (rivoric) [reply templates/sql-server.json]: This still appears to be there

Comment (qamago) [reply templates/sql-server.json]: removed

**Pull request merged**